Mention hosting bundle for 500 internals (#24138)
dotnet/AspNetCore.Docs: aspnetcore/test/troubleshoot-azure-iis.md

This error occurs within the app's code during startup or while creating a response. The response may contain no content, or the response may appear as a *500 Internal Server Error* in the browser. The Application Event Log usually states that the app started normally. From the server's perspective, that's correct. The app did start, but it can't generate a valid response. Run the app at a command prompt on the server or enable the ASP.NET Core Module stdout log to troubleshoot the problem.

> This error also may occur when the hosting bundle is not installed or is corrupted. It maybe worth trying to install or repair the installation of the .NET Core Hosting Bundle (for IIS) or Visual Studio (for IIS Express).

Reviewer: ```suggestion
This error also may occur when the hosting bundle is not installed or is corrupted. Installing or repairing the installation of the .NET Core Hosting Bundle (for IIS) or Visual Studio (for IIS Express) may fix the problem.
```
